apps/chat/src/chat/chat.service.spec.ts

```diff
@@ -61,6 +114,7 @@ describe('ChatService', () => {
   it('should be defined', () => {
```

Comment:
```suggestion
  it('Chat Service, Model이 정의된다.', () => {
```
위의 표현으로 변경 되도 좋을 것 같습니다!

Reply:
넵 반영하겠습니다!